aiokafka/conn.py

```diff
@@ -565,15 +679,16 @@ async def _read(self_ref):
             self._handle_frame(resp)
             del self
 
-    def _handle_frame(self, resp):
-        correlation_id, request, fut = self._requests[0]
+    def _handle_frame(self, resp: bytes) -> None:
+        packet = self._requests[0]
 
-        if correlation_id is None:  # Is a SASL packet, just pass it though
-            if not fut.done():
-                fut.set_result(resp)
+        if packet.correlation_id is None:  # Is a SASL packet, just pass it though
```

Comment:
There is not need to define meaningless fields in `SaslPacket`, if we use `isinstance` here

Reply:
```python
class SaslPacket(NamedTuple):
    fut: asyncio.Future
```
without correation_id and request?